src/ekat/util/ekat_test_utils.hpp

```diff
@@ -72,6 +60,15 @@ void catch2_req_pk_sensitive(const Scalar lhs, const Scalar rhs)
   }
 }
 
+// Check whether a string (from argv) mathces a predefined option name,
```

Comment:
mathces -> matches (not a big deal... in case you're wondering, I was a writer for a couple of years, so my eyes catch this stuff almost automatically.)